--- origsrc/oxygen-gtk-1.0.0/src/CMakeLists.txt	2010-12-12 07:55:34.000000000 -0600
+++ src/oxygen-gtk-1.0.0/src/CMakeLists.txt	2010-12-26 23:30:06.218000800 -0600
@@ -45,7 +45,7 @@ set( OXYGEN-GTK-SOURCES
     animations/oxygentreeviewengine.cpp
  )
 
-add_library( oxygen-gtk SHARED ${OXYGEN-GTK-SOURCES} )
+add_library( oxygen-gtk MODULE ${OXYGEN-GTK-SOURCES} )
 
 # Add GTK's cflags
 set( CMAKE_CXX_FLAGS " -Wall ${CMAKE_CXX_FLAGS} ${GTK_CFLAGS} -g" )
